src/deck/backends/Kbuild

```diff
@@ -0,0 +1,2 @@
+obj-y += deck_backend_onewire.o
+obj-y += deck_backend_forced.o
```

Comment:
Since these are implemented dynamically, discovered by linker script, what about making the forced deck backend optional and only active when there is a deck to force.